ONSdigital/eq-survey-runner
# Restructure static assets to have a clear project structure

### What is the context of this PR?

This PR restructures the static assets to a cleaner folder structure so that
the top level of the application is easier to understand.

It places all static assets (CSS, JS, IMGS) in a single asset folder and also changes any paths in the tests to ensure they still all pass.
### How to review
1. Check out this branch
2. Delete any local static folders (the temporary asset folder created by the asset pipeline)
3. Run the app locally.
4. Walk through one of the surveys, looking for any 404's in the browser console or anything that looks weird (fonts, behaviours, etc)
5. Run the tests and check that all of the tests correctly pass.
### Who worked on the PR

@dhilton - probably best that @bennpearson or @hamishtaplin review this change.
